task8/index.js

```diff
@@ -0,0 +1,49 @@
+const obj = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+const div = document.getElementById("js-parent");
+
+const loading = () => {
+    const loadingPlace = document.createElement("div");
+    const gif = document.createElement('img');
+    loadingPlace.id = "loadingPlace";
+    gif.src = "loading-circle.gif";
+    div.appendChild(loadingPlace).appendChild(gif);
+};
+
+const loaded = () => {
+    const loadingPlace = document.getElementById("loadingPlace");
+    loadingPlace.remove();
+}
```

Comment:
```suggestion
const removeLoading = () => {
    document.getElementById("loadingPlace").remove()
}
```

Reply:
It's going to be so short!
Thank you for telling me.